Project_Charter.markdown

```diff
@@ -110,8 +108,11 @@ to be satisfied.
 
 ## Scope
```

Comment:
Do we want to put scope for MVP and full project separate or together?
and
I was just wondering if we could just separate our points into two sections: 
inclusive boundaries: 
exclusive boundaries:
- Video calls are not allowed.
- Will not supply the servers, hardware or OS
- Integration with other third party app is not supported.